### ISSUE TITLE
Voiceover doesn't read chats in languages with partial localization

### ISSUE THREAD

**Issue by roostr** [jessesquires/JSQMessagesViewController#1903]
## New issue checklist

- [x] I have read all of the [`README`](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/README.md), [documentation](http://cocoadocs.org/docsets/JSQMessagesViewController/), and [FAQ](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/Documentation/faq.md).
- [X] I have reviewed the [contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md). Confirmation: 💪😎👊
- [X] I have searched [existing issues](https://github.com/jessesquires/JSQMessagesViewController/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

## General information

Opening this issue to submit a PR to fix it.

The library is localized into many languages, but only some have all 7 localization keys:

- "load_earlier_messages"
- "send"
- "new_message"
- "text_message_accessibility_label"
- "media_message_accessibility_label"
- "accessory_button_accessibility_label"
- "new_message_received_accessibility_announcement"

Others have 3 localization keys:

- "load_earlier_messages"
- "send"
- "new_message"

The issue is that if iOS finds a localization file, but it doesn't find a key in it for the value you're looking up, then it does not look in the base localization by default. Thus, for all the accessibility keys, lookup fails and it returns the key as the value.

The result of this is when you're using a language such as Italian (has partial translation), and you're reading the message bubble stream with Voiceover, it reads the following for each chat bubble: "text_message_accessibility_label", "text_message_accessibility_label", "text_message_accessibility_label", etc.

The patch I'm submitting will properly fall back on the string in the base translation if a partial localization exists, but the key being looked up does not exist in that localization.

**Comment by roostr** [comment]
Steps to reproduce:

First, you'll need a demo app is localized to an affected language, such as Italian. To do this, we need to make sure at least one file in the Italian localization for the demo app is localized.

1. In XCode, select the JSQMessages demo project
2. Under the "Info" header, under "Localizations", find the "Italian" language. It should say "0 Files Localized"
3. Delete the Italian localization
4. Add the Italian localization,  with "Main.storyboard" checked.
5. Push this build to a phone.

![screen shot 2016-11-19 at 1 44 27 pm](https://cloud.githubusercontent.com/assets/978146/20457699/a18a521e-ae5e-11e6-8297-c2dd926222a4.png)

Now, we need to perform Voiceover on a chat conversation:

1. Settings app > General > Accessibility > Accessibility Shortcut > VoiceOver (turn on)
2. Settings app > General > Language & Region > iPhone Language > Italian
3. Launch the JSQMessages demo app
4. Select "Push via storyboard"
5. Triple-tap home button (turn VoiceOver on)
6. Tap the different chat bubbles

Expected result:
 * VoiceOver reads the name of the contact and the message in an Italian accent.

Actual result:
 * VoiceOver reads "text_message_accessibility_label" in an Italian accent.

**Comment by jessesquires** [comment]
Hello everyone!

I'm sorry to inform the community that I'm officially deprecating this project. 😢 Please read my blog post for details:

http://www.jessesquires.com/blog/officially-deprecating-jsqmessagesviewcontroller/

Thus, I'm closing all issues and pull requests and making the necessary updates to formally deprecate the library. I'm sorry if this is unexpected or disappointing. Please know that this was an extremely difficult decision to make. I'd like to thank everyone here for contributing and making this project so great. It was a fun 4 years. 😊 

Thanks for understanding,
— jsq
